cmd/notary/tuf.go

```diff
@@ -447,3 +448,38 @@ func getRemoteTrustServer() string {
 	}
 	return remoteTrustServer
 }
+
+type targetsSorter []*notaryclient.Target
+
+func (t targetsSorter) Len() int      { return len(t) }
+func (t targetsSorter) Swap(i, j int) { t[i], t[j] = t[j], t[i] }
+func (t targetsSorter) Less(i, j int) bool {
+	return t[i].Name < t[j].Name
+}
+
+// Given a list of KeyStores in order of listing preference, pretty-prints the
+// root keys and then the signing keys.
+func prettyPrintTargets(ts []*notaryclient.Target, writer io.Writer) {
+	if len(ts) == 0 {
+		writer.Write([]byte("No targets present in this repository.\n"))
+	}
+
+	sort.Stable(targetsSorter(ts))
+
+	table := tablewriter.NewWriter(writer)
+	table.SetHeader([]string{"Name", "Digest", "Size (bytes)"})
+	table.SetBorder(false)
+	table.SetColumnSeparator(" ")
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
```

Comment:
Non-blocking - I vaguely remember a slack discussion about whether this should be centered?  If so, should this line be removed (IIRC tablewriter is by default centered, but could also specify a center alignment)